packages/hash/frontend/src/components/layout/PageHeader/SearchBar/SearchInput.tsx

```diff
@@ -130,76 +130,50 @@ export const SearchInput: React.FC<{
+      type="text"
+      value={displayedQuery}
+      onFocus={() => setResultListVisible(true)}
+      onChange={(event) => {
+        setResultListVisible(true);
+        setQueryText(event.target.value);
+      }}
+      sx={{
         width: isMobile ? "100%" : "385px",
-        ":focus-within": {
-          margin: "-1px",
-          borderWidth: 2,
-          borderColor: theme.palette.blue[70],
-          transition: theme.transitions.create([
-            "border-color",
-            "border-width",
-            "margin",
-          ]),
-          ".MuiInputAdornment-root": {
-            marginRight: "-2px",
-            transition: theme.transitions.create(["margin"]),
-          },
+        "& .MuiInputBase-input": {
+          paddingLeft: "unset",
+          paddingRight: isMobile ? 1 : "unset",
         },
-      })}
-    >
-      <Box
-        sx={(theme) => ({
-          padding: theme.spacing(0, 1.5),
-          height: "100%",
-          position: "absolute",
-          pointerEvents: "none",
-          display: "flex",
-          alignItems: "center",
-          justifyContent: "center",
-        })}
-      >
-        <SearchIcon sx={{ height: "16px", width: "auto" }} />
-      </Box>
-      <InputBase
-        placeholder="Search for anything"
-        inputRef={inputRef}
-        type="text"
-        value={displayedQuery}
-        onFocus={() => setResultListVisible(true)}
-        onChange={(event) => {
-          setResultListVisible(true);
-          setQueryText(event.target.value);
-        }}
-        sx={(theme) => ({
-          color: "inherit",
-          width: "100%",
-          lineHeight: "18px",
-          "& .MuiInputBase-input": {
-            py: theme.spacing(1),
-            paddingLeft: theme.spacing(4.5),
-            paddingRight: isMobile ? 1 : "unset",
-          },
-        })}
-        inputProps={{ "aria-label": "search" }}
-        endAdornment={
-          !isMobile ? (
-            <InputAdornment position="end">
-              {displayedQuery ? (
-                <ClearSearchIcon clearSearch={clearSearch} />
-              ) : (
-                <ShortcutIcon />
-              )}
-            </InputAdornment>
-          ) : null
-        }
-      />
-    </Box>
+      }}
+      inputProps={{ "aria-label": "search" }}
+      startAdornment={
+        <InputAdornment position="start">
+          <Box
+            sx={{
+              height: "100%",
+              pointerEvents: "none",
+              display: "flex",
+              alignItems: "center",
+              justifyContent: "center",
+            }}
+          >
+            <SearchIcon sx={{ height: "16px", width: "auto" }} />
+          </Box>
+        </InputAdornment>
+      }
+      endAdornment={
+        !isMobile ? (
+          <InputAdornment position="end">
+            {displayedQuery ? (
+              <ClearSearchIcon clearSearch={clearSearch} />
+            ) : (
+              <ShortcutIcon />
```

Comment:
**Blocking:** This shortcut icon has being squished, creating an oval outline/hover state:
<img width="47" alt="image" src="https://user-images.githubusercontent.com/42802102/158164899-7dd5689c-ccb3-43d8-ae5e-4bcdcf90646c.png">
<img width="55" alt="image" src="https://user-images.githubusercontent.com/42802102/158165018-b9a8c10a-f052-44d2-a1a1-66e3faa6bb60.png">

Reply:
Done here: https://github.com/hashintel/hash/pull/375/commits/f54f760153097baa2dd30892dc935974300462fd#diff-f6666bfe47541e94befe2f523dbbc125b1e02c578a1357d8e56a70b0d45a1bc5R199